packages/db/src/repositories/boost.ts

```diff
@@ -641,7 +642,7 @@ class BoostRepository {
     return await executor
       .select({
         userId: schema.boostBalances.userId,
-        wallet: schema.boostChanges.wallet,
```

Comment:
fetching the wallet address from user instead of boost_changes because will stop using the wallet from boost_changes